### PR TITLE
Feature/eea

### PR DESCRIPTION
**Pull request by iikirilov**
### What does this PR do?
Adds EEA module to interface with EEA compliant client. The implemented methods are
`eea_sendRawTransaction`
`eea_getTransactionReceipt`

### Where should the reviewer start?
The eea module

### Why is it needed?
To be able to interface with the EEA endpoints in the Pantheon client

### CODE REVIEW

File: eea/build.gradle [hunk 1]
```diff
@@ -0,0 +1,7 @@
+
+description 'Eea JSON-RPC API'
+
```
Comment (antonydenyer) [inline eea/build.gradle]: Nit `EEA`

**Comment by iikirilov** [comment]
There is an update coming to this later today

**Comment by iikirilov** [comment]
This is ready now

**Comment by iikirilov** [comment]
This is ready now

**Comment by iikirilov** [comment]
Did some playing around - there is another bug - please do not merge

**Comment by iikirilov** [comment]
Let's get this merged - sample project available here https://github.com/web3j/eea-sample-project to run against pegasyseng/pantheon-quickstart dockerized privacy environment

File: core/src/main/java/org/web3j/tx/response/PollingTransactionReceiptProcessor.java [hunk 2]
```diff
@@ -34,15 +34,17 @@ private TransactionReceipt getTransactionReceipt(
             throws IOException, TransactionException {
 
         Optional<TransactionReceipt> receiptOptional =
-                sendTransactionReceiptRequest(transactionHash);
+                (Optional<TransactionReceipt>) sendTransactionReceiptRequest(transactionHash);
```
Comment (snazha-blkio) [inline core/src/main/java/org/web3j/tx/response/PollingTransactionReceiptProcessor.java]: @iikirilov  can you please check if you could use `Optional.ofNullable()`?

File: core/src/main/java/org/web3j/tx/response/PollingTransactionReceiptProcessor.java [hunk 3]
```diff
@@ -34,15 +34,17 @@ private TransactionReceipt getTransactionReceipt(
...
         for (int i = 0; i < attempts; i++) {
             if (!receiptOptional.isPresent()) {
                 try {
                     Thread.sleep(sleepDuration);
                 } catch (InterruptedException e) {
                     throw new TransactionException(e);
                 }
-                receiptOptional = sendTransactionReceiptRequest(transactionHash);
+                receiptOptional =
+                        (Optional<TransactionReceipt>)
```
Comment (snazha-blkio) [inline core/src/main/java/org/web3j/tx/response/PollingTransactionReceiptProcessor.java]: @iikirilov  can you please check if you could use `Optional.ofNullable()`?

**Pull request merged**